docs/dev/how-to-develop-using-perl.md

```diff
@@ -0,0 +1,48 @@
+# How can I learn the Perl programming language?
+
+Here are some introductory resources to learn Perl:
+
+## Quick start 
+
+- [Perl Youtube Tutorial](https://www.youtube.com/watch?v=c0k9ieKky7Q) - Perl Enough to be dangerous // FULL COURSE 3 HOURS.
+- [Perl - Introduction](https://www.tutorialspoint.com/perl/perl_quick_guide.htm) - Introduction to perl from tutorialspoint
+- [Impatient Perl](https://blob.perl.org/books/impatient-perl/iperl.pdf) - PDF document for people wintrested in learning perl.
+
+## Official Documentation
+
+- [Perl.org](https://www.perl.org/) - Official Perl website with documentation, tutorials, and community resources.
+- [Learn Perl](https://learn.perl.org/) - Perl programming language tutorials for beginners.
+- [Perl Maven](https://perlmaven.com/) - Perl programming tutorials, tips, and code examples.
+
+# See the logs while running Perl locally
```

Comment:
Great doc. 

But I would have made it a specific how to, and a "how-to-develop-using-perl" that links to how to learn perl, the how to use log, and [how-to-write-and-run-tests.md](https://github.com/openfoodfacts/openfoodfacts-server/blob/main/docs/dev/how-to-write-and-run-tests.md), [how-to-use-repl.md](https://github.com/openfoodfacts/openfoodfacts-server/blob/main/docs/dev/how-to-use-repl.md) and [how-to-use-vscode.md](https://github.com/openfoodfacts/openfoodfacts-server/blob/main/docs/dev/how-to-use-vscode.md)